packages/block-editor/src/components/contrast-checker/index.native.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { Text } from 'react-native';
+import tinycolor from 'tinycolor2';
```

Comment:
It looks like the [tinycolor](https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+tinycolor) usage was replaced in favor of `colord`. Do you think we could use it as well for this feature?

Reply:
Ah, thank you for spotting that! It looks like `colord` was introduced after I began work on this and I didn't notice. I updated the references in https://github.com/WordPress/gutenberg/pull/34902/commits/215f95cc9bf6d207cfe6532729b7d419fb71e3b7.